lib/parser.js

```diff
@@ -347,6 +347,12 @@ module.exports = function parser(identifier, string) {
         expect(Token.Type.END_PAREN)
       }
 
+      if (peek().type == Token.Type.DOT) {
+        expect(Token.Type.DOT)
+        token = expect(Token.Type.WORD)
+        name = token.content
```

Comment:
so, not setting `token.content` to `name` here means that `name` becomes `options['other.field'] = 'special'`

i think do something like you suggest would require adding some logic to create an object for options that are "nested" like this.